platforms/jvm/language-java/src/test/groovy/org/gradle/api/internal/tasks/compile/DiagnosticToProblemListenerTest.groovy

```diff
@@ -30,7 +30,7 @@ class DiagnosticToProblemListenerTest extends Specification {
         1 * additionalData(org.gradle.api.problems.internal.GeneralDataSpec, _)
     }
 
-    def diagnosticToProblemListener = new DiagnosticToProblemListener(null, null)
+    def diagnosticToProblemListener = new DiagnosticToProblemListener(null, null as Context)
```

Comment:
🤔 Since we have only one constructor now, we probably don't need to cast on the second parameter.